custom_components/easee/services.py

```diff
@@ -3,15 +3,16 @@
 import logging
 from homeassistant.helpers import config_validation as cv
 from homeassistant.exceptions import HomeAssistantError
+from homeassistant.util import dt as dt
 
 from .const import DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
 CHARGER_ID = "charger_id"
 CIRCUIT_ID = "circuit_id"
-ATTR_CHARGEPLAN_START_TIME = "chargeStartTime"
-ATTR_CHARGEPLAN_STOP_TIME = "chargeStopTime"
+ATTR_CHARGEPLAN_START_TIME = "start_time"
```

Comment:
If this is an iso datetime string I think it is better to call it "start_date_time"
Same with stop_time. I know easee uses chargeStartTime but that is also a bad name.

Reply:
Agreed. Will fix.